test/aqs_test_module.yaml

```diff
@@ -0,0 +1,96 @@
+# Simple test spec
```

Comment:
The aqs stuff actually needs a project like this for its v1 api in any case. Do you think we could use this for the real thing?